### PR TITLE
fix(challenge): check inline style for color

### PR DESCRIPTION
**Pull request by lasjorg**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Fix for the challenge: https://www.freecodecamp.org/learn/responsive-web-design/basic-css/change-the-color-of-text

Switch the test to look at the style property for the color value to avoid issues with extensions that change the colors (like dark mode extensions).

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-css/change-the-color-of-text.english.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ tests:
   - text: Your <code>h2</code> element should have a <code>style</code> declaration.
     testString: assert($("h2").attr('style'));
   - text: Your <code>h2</code> element should be red.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/01-responsive-web-design/basic-css/change-the-color-of-text.english.md]:
```suggestion
  - text: Your <code>h2</code> element should have the color <code>red</code>.
```
If we're only going to accept `color: red` I think there should be a stronger hint that that's the case.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/01-responsive-web-design/basic-css/change-the-color-of-text.english.md (hunk 1 above)
Comment (thecodingaviator) [reply curriculum/challenges/english/01-responsive-web-design/basic-css/change-the-color-of-text.english.md]: How about this @ojeytonwilliams ?
```suggestion
  - text: Your <code>h2</code> element should have the color set to <code>red</code>.
```

Comment (ojeytonwilliams) [reply curriculum/challenges/english/01-responsive-web-design/basic-css/change-the-color-of-text.english.md]: Yeah, that works for me.

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Please make the requested changes

**Review by ojeytonwilliams — APPROVED** [review]

**Review by thecodingaviator — APPROVED** [review]

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**